package.json

```diff
@@ -63,14 +63,14 @@
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-react": "^7.12.10",
     "@types/classnames": "^2.3.1",
+    "@types/electron": "^1.6.10",
     "@types/jest": "^26.0.22",
     "@types/react": "^17.0.5",
     "@types/sprintf-js": "^1.1.2",
     "@typescript-eslint/eslint-plugin": "^4.22.0",
     "@typescript-eslint/parser": "^4.22.0",
     "ajv": "^5.5.2",
     "babel-loader": "^8.1.0",
-    "electron": "11.3.0",
```

Comment:
If electron is not in "devDependencies", we will encounter error below when building:
"electron" is only allowed in "devDependencies" stackoverflow electron builder